debian/control needs to list binutils-<nativetriplet>:<nativearch> for each
relevant architecture. Since new ports are generally not known to stable dpkg,
listing the relevant binutils-<nativetriplet> package in debian/control would
get the package rejected by dak. Thus we cannot hope to provide a complete list
of architectures and instead provide a hand curated list where the build should
just work.

Index: debian/rules
===================================================================
--- debian.orig/rules
+++ debian/rules
@@ -81,6 +81,14 @@
   endif
 endif
 
+NATIVE_ARCHS ?= alpha amd64 arm64 armel armhf hppa hurd-i386 i386 ia64 \
+		kfreebsd-amd64 kfreebsd-i386 m68k \
+		mips mipsel mipsn32 mipsn32el \
+		mipsn32r6 mipsn32r6el mipsr6 mipsr6el \
+		mips64 mips64el mips64r6 mips64r6el \
+		nios2 or1k powerpc powerpcspe ppc64 ppc64el s390 s390x sh4 \
+		sparc sparc64 x32
+
 src_name := $(shell dpkg-parsechangelog -S Source)
 ifeq ($(src_name),binutils)
   with_native	= yes
